src/TropicalGemmFP32.cu

```diff
@@ -0,0 +1,627 @@
+// This CUDA code is modified based on github repo https://github.com/Yinghan-Li/YHs_Sample, which is under GPL 3.0 License
```

Comment:
Holy, the GPL3 license, that is sexy. If we decide to keep this version in our code base, we have to include GPL3 license.
> To “propagate” a work means to do anything with it that, without permission, would make you directly or secondarily liable for infringement under applicable copyright law, except executing it on a computer or modifying a private copy. Propagation includes copying, distribution (with or without modification), making available to the public, and in some countries other activities as well.